nncf/torch/quantization/reference.py

```diff
@@ -121,5 +122,30 @@ def tune_range(
 
 class ReferenceQuantizedFunctions:
     _executor = ReferenceQuantize(backend_type=ReferenceBackendType.TORCH)
-    Quantize_forward = _executor.forward
-    Quantize_backward = _executor.backward
+    try:
+
+        def sample():
+            return torch.tensor([1, 1]).to(torch.float32)
+
+        Quantize_forward = torch.compile(_executor.forward)
+        Quantize_backward = torch.compile(_executor.backward)
```

Comment:
Is it possible to move this check outside of class on the module level?

Reply:
Done